application/models/services/ThemeQuestionAttributeProvider.php

```diff
@@ -53,8 +53,32 @@ protected function getAttributesFromQuestionTheme($questionThemeName, $questionT
             }
             // Create array of attribute with name as key
             foreach ($xmlAttributes['attribute'] as $attribute) {
-                if (!empty($attribute['name'])) {
-                    $attributes[$attribute['name']] = array_merge(self::getBaseDefinition(), $attribute);
+                if (empty($attribute['name'])) {
+                    /* Allow comments in attributes */
+                    continue;
+                }
+                /* settings the default value */
+                $attributes[$attribute['name']] = self::getBaseDefinition();
+                /* settings the xml value */
+                foreach ($attribute as $property => $propertyValue) {
```

Comment:
It fix some other things ? No ?

Reply:
This code was present for the core question themes. Now is also available for custom ones.